import sys
import ctypes
import logcat

print('hello')
raise ValueError('this is an exception')